com.github.philip_scott.spice-up.json

```diff
@@ -0,0 +1,95 @@
+{
+    "id": "com.github.philip_scott.spice-up",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.24",
+    "sdk": "org.gnome.Sdk",
+    "command": "com.github.philip-scott.spice-up",
+    "rename-icon": "com.github.philip-scott.spice-up",
+    "rename-desktop-file": "com.github.philip-scott.spice-up.desktop",
+    "rename-appdata-file": "com.github.philip-scott.spice-up.appdata.xml",
+    "finish-args": [
+            "--share=ipc", "--socket=x11",
+            "--device=all",
+            "--socket=wayland",
+            "--filesystem=home",
+            /* this is bad but doesn't work otherwise */
+            "--socket=session-bus",
```

Comment:
What's the error you get? You probably need to use `--talk-name=` or `--own-name=` to narrow this down.

Reply:
He needs to patch this to use the correct app-id: https://github.com/Philip-Scott/Spice-up/blob/master/src/Application.vala#L37

Reply:
Thanks! Makes perfect sense. Can't I just own the old name on the bus? Would save a source code patch and leave us closer to an unmodified upstream source. I've confirmed that works.

Reply:
Well, I'm not sure honestly. If you rename it we get the clean situation where the flatpak only owns the name you expect it to. There is value in matching upstream but we already broke that by renaming it elsewhere.